rz-sbc/rzsbc_yocto.sh

```diff
@@ -242,6 +243,39 @@ function build() {
     #output
 }
 
+# Main build-sdk
+function build-sdk() {
+    setup $1
+
+    # Build RZ
+    cd ${RZ_TARGET_DIR}
+    echo "In yocto. pwd = ${PWD}"
+    #source poky/oe-init-build-env
+    echo "Env setup completed. pwd = ${PWD}"
+
+    # Legacy style
+    #cp ../meta-renesas/docs/template/conf/rzpi/* conf/
+    #bitbake core-image-qt
+
+    # New style
+    TEMPLATECONF=$PWD/meta-renesas/meta-rzg2l/docs/template/conf/rzpi source poky/oe-init-build-env build
+
+    # Copy overrides file as yocto doesnt copy site.conf.sample
+    cp ../meta-renesas/meta-rzg2l/docs/template/conf/rzpi/site.conf.sample conf/site.conf
+
+    # Initiate build
+    MACHINE=rzpi bitbake core-image-qt
```

Comment:
The two commands in lines 264 and 267 will make no sense when it's already a build. Make sure you separate two cases:
- Build SDK from scratch.
- Build SDK after a common build.